sklearn/tree/export.py

```diff
@@ -362,14 +370,23 @@ def recurse(tree, node_id, criterion, parent=None, depth=0):
                 out_file.write('%d -> %d ;\n' % (parent, node_id))
 
     own_file = False
+    return_string = False
     try:
+        if out_file == 'tree.dot':
+            warnings.warn("out_file can be set to None from 0.18, "
```

Comment:
`out_file can be set to None starting from 0.18. This will be the default in 0.20` might be more clear